microsoftgraph/microsoft-graph-explorer-v4
# Task: Add php snippets tab

## Overview
Closes #1922 

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
Click on the Code snippets tab
Click on PHP snippets tab
Notice that PHP snippets are available
